src/main/java/org/candlepin/subscriptions/metering/service/prometheus/MetricProperties.java

```diff
@@ -70,68 +80,9 @@
      */
     private double backOffMultiplier = 2;
 
-    public String getMetricPromQL() {
-        return metricPromQL;
-    }
-
-    public void setMetricPromQL(String metricPromQL) {
-        this.metricPromQL = metricPromQL;
-    }
-
-    public int getStep() {
-        return step;
-    }
-
-    public void setStep(int step) {
-        this.step = step;
-    }
-
-    public int getRangeInMinutes() {
-        return rangeInMinutes;
-    }
-
-    public void setRangeInMinutes(int rangeInMinutes) {
-        this.rangeInMinutes = rangeInMinutes;
-    }
-
-    public int getQueryTimeout() {
-        return queryTimeout;
-    }
-
-    public void setQueryTimeout(int queryTimeout) {
-        this.queryTimeout = queryTimeout;
-    }
-
-    public int getMaxAttempts() {
-        return maxAttempts;
-    }
-
-    public void setMaxAttempts(int maxAttempts) {
-        this.maxAttempts = maxAttempts;
-    }
-
-    public long getBackOffMaxInterval() {
-        return backOffMaxInterval;
-    }
-
-    public void setBackOffMaxInterval(long backOffMaxInterval) {
-        this.backOffMaxInterval = backOffMaxInterval;
-    }
-
-    public long getBackOffInitialInterval() {
-        return backOffInitialInterval;
-    }
-
-    public void setBackOffInitialInterval(long backOffInitialInterval) {
-        this.backOffInitialInterval = backOffInitialInterval;
-    }
-
-    public double getBackOffMultiplier() {
-        return backOffMultiplier;
-    }
-
-    public void setBackOffMultiplier(double backOffMultiplier) {
-        this.backOffMultiplier = backOffMultiplier;
-    }
+    /**
+     * Batch size to use while persisting events.
+     */
+    private int eventBatchSize = 1000;
 
```

Comment:
I'm confused by a recent merge conflict.  Is eventBatchSize supposed to exist here, or did I add it back by mistake?

Reply:
Looks like accidental reintroduction.